### PR TITLE
Spot check for broken asterisks and image tag - Dashboards

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/dashboards/about.md [hunk 1]
```diff
@@ -92,26 +92,17 @@ Live mode restrictions do not apply to Dashboard. 
...
-There are some restrictions when using operators with dashboards. To learn more, see [Restricted Operators in Dashboards](/docs/dashboards/restricted-operators-dashboards).
-
-![auto refresh dropdown options.png](/img/dashboards-new/about-dashboard-new/auto-refresh-dropdown-options.png)
-
-A list of the refresh interval rates is provided for you to select from.
-
-![dashboard new refresh interval options.png](/img/dashboards-new/about-dashboard-new/dashboard-new-refresh-interval-options.png)
+There are some restrictions when using operators with dashboards. To learn more, see [Restricted Operators in Dashboards](/docs/dashboards/restricted-operators-dashboards).<br/><img src={useBaseUrl('/img/dashboards-new/about-dashboard-new/auto-refresh-dropdown-options.png')} alt="auto refresh dropdown options" style={{border: '1px solid gray'}} width="600" />
 
+A list of the refresh interval rates is provided for you to select from.<br/><img src={useBaseUrl('/img/dashboards-new/about-dashboard-new/dashboard-new-refresh-interval-options.png')} alt="dashboard new refresh interval options" style={{border: '1px solid gray'}} width="600" />
```
Comment (jpipkin1) [inline docs/dashboards/about.md:97]:
```suggestion
A list of the refresh interval rates is provided for you to select from.<br/><img src={useBaseUrl('/img/dashboards-new/about-dashboard-new/dashboard-new-refresh-interval-options.png')} alt="dashboard new refresh interval options" style={{border: '1px solid gray'}} width="200" />
```

File: docs/dashboards/faq.md [hunk 2]
```diff
@@ -55,12 +56,8 @@ The types of charts you will find in the time series panel include:
 
 ## How do I overlay logs and metrics on one panel?
 
-One of the valuable features in dashboards is that you can graph data on to the same panel across data sources very easily. Inside of each dashboard panel, you have the option of having up to 6 metrics and 6 logs query rows. To create a panel with both logs and metrics derived data, press the **Add a Panel** button, select **Time Series**, and you will launch the panel builder.
+One of the valuable features in dashboards is that you can graph data on to the same panel across data sources very easily. Inside of each dashboard panel, you have the option of having up to 6 metrics and 6 logs query rows. To create a panel with both logs and metrics derived data, press the **Add a Panel** button, select **Time Series**, and you will launch the panel builder.<br/><img src={useBaseUrl('/img/dashboards-new/dashboard-new-faqs/Add-a-panel-button.png')} alt="Add a panel button" style={{border: '1px solid gray'}} width="400" />
```
Comment (jpipkin1) [inline docs/dashboards/faq.md:59]:
```suggestion
One of the valuable features in dashboards is that you can graph data on to the same panel across data sources very easily. Inside of each dashboard panel, you have the option of having up to 6 metrics and 6 logs query rows. To create a panel with both logs and metrics derived data, press the **Add a Panel** button, select **Time Series**, and you will launch the panel builder.<br/><img src={useBaseUrl('/img/dashboards-new/dashboard-new-faqs/Add-a-panel-button.png')} alt="Add a panel button" style={{border: '1px solid gray'}} width="600" />
```

File: docs/dashboards/faq.md [hunk 3]
```diff
@@ -55,12 +56,8 @@ The types of charts you will find in the time series panel include:
...
 
-![Add a panel button.png](/img/dashboards-new/dashboard-new-faqs/Add-a-panel-button.png)
-
-To switch between a metrics query row and a logs query row, you just have to select the Logs query dropdown after the **#A** and change it to Metrics. To add multiple logs or metrics rows, hover over the last query row and press the + button.
-
-![Logs and metrics query options.png](/img/dashboards-new/dashboard-new-faqs/Logs-and-metrics-query-options.png)
+To switch between a metrics query row and a logs query row, you just have to select the Logs query dropdown after the **#A** and change it to Metrics. To add multiple logs or metrics rows, hover over the last query row and press the + button.<br/><img src={useBaseUrl('/img/dashboards-new/dashboard-new-faqs/Logs-and-metrics-query-options.png')} alt="Logs and metrics query options" style={{border: '1px solid gray'}} width="400" />
```
Comment (jpipkin1) [inline docs/dashboards/faq.md:61]:
```suggestion
To switch between a metrics query row and a logs query row, you just have to select the Logs query dropdown after the **#A** and change it to Metrics. To add multiple logs or metrics rows, hover over the last query row and press the + button.<br/><img src={useBaseUrl('/img/dashboards-new/dashboard-new-faqs/Logs-and-metrics-query-options.png')} alt="Logs and metrics query options" style={{border: '1px solid gray'}} width="600" />
```

**Review by jpipkin1 — APPROVED** [review]
Nice work, @JV0812! I just have a few suggestions for image sizing.

**Comment by JV0812** [comment]
Thanks @jpipkin1 for your review.

**Pull request merged**